backend/app/api/routes/model_management.py

```
@@ -0,0 +1,263 @@
+
+class ModelOperationResponse(BaseModel):
+    """模型操作响应结构"""
+    success: bool
+    message: str
+    model_name: Optional[str] = None
+
+
+async def _call_litellm_admin_api(
+    method: str,
+    endpoint: str,
+    data: Optional[Dict[str, Any]] = None
```

Ruff (0.11.9)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)